cucumber/authenticators_jwt/features/step_definitions/authn_jwt_steps.rb

```diff
@@ -6,6 +6,10 @@
   create_jwt_secret(variable, value)
 end
 
+Given(/I successfully set authn-jwt provider-uri variable with value of "([^"]*)" endpoint/) do |provider_uri|
```

Comment:
Delete this step and use the one in line 5 here

Reply:
nice catch